tests/e2e/e2e_suite_test.go

```diff
@@ -0,0 +1,85 @@
+package e2e_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/rancher-sandbox/rancheros-operator/tests/catalog"
+	"github.com/rancher-sandbox/rancheros-operator/tests/sut"
+)
+
+var (
+	chart      string
+	externalIP string
+)
+
+func TestE2e(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "os2 e2e test Suite")
+}
+
+var _ = BeforeSuite(func() {
+
+	kubectl := sut.NewKubectl()
+
+	chart = os.Getenv("ROS_CHART")
+	if chart == "" {
+		Fail("No chart provided, skipping tests")
```

Comment:
Skip instead of Fail

Reply:
Well, tests depends on it!

Reply:
oh well, then we should change the text :D